lib/templates/roxml.erb

```diff
@@ -5,7 +5,7 @@ require "roxml"
 require "<%= mod %>"
 <%- end -%>
 
-<%= @class.module.map { |mod| "module #{mod}" }.join " " %>
+<%= @class.module.map { |mod| "module #{mod}" }.join "; " %>
```

Comment:
The request was made to notify you about the issue, originally, this line was not planned to be included, the coding style I use requires the ";" here, just skip it if you did not find this right.